MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```

Comment:
- [OPTIONAL] Please ensure customizing the MIT file by adding your name at the top and changing the text [APP TYPE] with either "Website" or "webpage"